CoreFoundation/NumberDate.subproj/CFDate.c

```diff
@@ -104,6 +116,9 @@ CF_EXPORT CFTimeInterval CFGetSystemUptime(void) {
         HALT;
     }
     return (double)res.tv_sec + ((double)res.tv_nsec)/1.0E9;
+#elif DEPLOYMENT_TARGET_WINDOWS
+    ULONGLONG ullTickCount = GetTickCount64();
+    return ullTickCount / 1000;
```

Comment:
Should this be floating-point division?

Reply:
No, its integral, the tick count is an integral value.